### DISCUSSION TITLE
bookmarks not downloading to be exported

### DISCUSSION THREAD

**Discussion by mrawbabylon** [bromite/bromite#1838] [Q&A]
hi it's kinda late so im sorry if this is poorly written. im transferring all my stuff to a new phone and finally got around to setting up bromite again. however my bookmarks arent downloading from my old phone so i cant send it. i click bookmarks, then that download icon, but the download doesn't show up in my downloads on bromite or in file explorer. please help me :(

p.s. im not a techie so please use simple terminology when explaining to me :((

**Comment by csagan5** [comment]
You need to submit an issue, it cannot be debugged or reproduced like this.

**Reply by mrawbabylon** [reply]
ok, i'll do that when i have some more time, thank you!!! 🥰
